### ISSUE TITLE
pyhull install on Windows 7 

### ISSUE THREAD

**Issue by falbriard** [materialsvirtuallab/pyhull#6]
Dear colleagues, 
 Getting an error when installing the pyhull package under Windows7 in a 64 bits version using the pip or easy_install. I´ve tried set the mingwg compiler in distutils.cfg, as well as useing Visual Studio runtime version 10, and I´ve also tried to run it inside a cygwin64 terminal. The error message is:

c:/mingw/bin/../lib/gcc/mingw32/4.8.1/../../../../mingw32/bin/ld.exe: build\temp.win-amd64-2.7\Release\src\_pyhull.o: bad reloc address 0x0 in section `.data'
collect2.exe: error: ld returned 1 exit status
error: Setup script exited with error: command 'gcc' failed with exit status 1

Is a tested build procedure availabe for Windows 7 64 bits?  Thanks in advance for your support. 
Regards, Claude

**Comment by Askateth** [comment]
@falbriard You, Sir, are a champ. I had to install first mingw32 (which is available at https://github.com/develersrl/gccwinbinaries to work with Python).

Then I downloaded PyHull 1.2.1 from here: https://pypi.python.org/pypi/pyhull/1.2.1

I decompressed the file, went into the command window on the decompressed directory, follow your command and it worked!!

(I say the above obvious steps just in case someone as dub as myself is interested).

I actually only need the in_convexhull function, which I am pretty confident I can develop by myself in simpy, but it would take a time I don't have, so this saved the day.
